### DISCUSSION TITLE
Class & Method name is not showing in skywalking ui

### DISCUSSION THREAD

**Discussion by vishalkadam18** [apache/skywalking#12148] [Q&A] [answered]
Hiii Team,

Is this features are in agent or not???

If yes, then why in skywalking ui is not showing.

Thanks,
Vishal.

**Comment by wu-sheng** [comment]
You need to be more specific, if you mean the SpringMVC plugin, there is a config to control that. Some plugins use class name and methods as operation name of a span, while others are not. Span concept doesn't bind with a specific method, so, your asking is not suitable in the real practice actually.

This is not a feature. This is about how codes are written.

**Reply by wu-sheng** [reply, marked as answer]
An additional information from `agent.config`.

```
# If true, the fully qualified method name will be used as the endpoint name instead of the request URL, default is false.
plugin.springmvc.use_qualified_name_as_endpoint_name=${SW_PLUGIN_SPRINGMVC_USE_QUALIFIED_NAME_AS_ENDPOINT_NAME:false}
```
